base-helm-configs/barbican/barbican-helm-overrides.yaml

```diff
@@ -74,7 +82,8 @@ conf:
       enable_proxy_headers_parsing: true
   barbican_api_uwsgi:
     uwsgi:
-      processes: 4
+      processes: 2
```

Comment:
Are these changes like these intended for the aio/converged/labs type env only?

Reply:
Got answer in standup and the answer is yes. These changes are intended for a lab/converged or otherwise smaller deployment.